libopenage/unit/action.cpp

```diff
@@ -10,10 +10,57 @@
 #include "action.h"
 #include "command.h"
 #include "producer.h"
+#include "research.h"
 #include "unit_texture.h"
 
 namespace openage {
 
+IntervalTimer::IntervalTimer(unsigned int interval)
+	:
+	IntervalTimer{interval, -1} {
+}
+
+IntervalTimer::IntervalTimer(unsigned int interval, int max_triggers)
+	:
+	interval{interval},
+	max_triggers{max_triggers},
+	time_left{interval},
+	triggers{triggers} {
```

Comment:
where does `triggers` come from? currently its the uninitialized member.

Reply:
Why can this even compile? It must obviously be zero